ibis/backends/flink/tests/test_ddl.py

```diff
@@ -427,3 +446,162 @@ def test_insert_simple_select(con, tempdir_sink_configs):
         temporary_file = next(iter(os.listdir(tempdir)))
         with open(os.path.join(tempdir, temporary_file)) as f:
             assert f.read() == '"fred flintstone",35\n"barney rubble",32\n'
+
+
+@pytest.mark.broken(
```

Comment:
also a minor comment: we might want to put these tests in a different file because join is not a DDL